Description: Filter non-Linux based native integrations from
 compliation using stapler.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: jenkins/core/pom.xml
===================================================================
--- jenkins.orig/core/pom.xml	2012-02-28 16:49:09.501797382 +0000
+++ jenkins/core/pom.xml	2012-02-28 16:49:10.609800755 +0000
@@ -587,6 +587,17 @@
         <configuration>
           <fork>${staplerFork}</fork><!-- the source code is big enough now that otherwise it fails with OutOfMemoryError -->
           <maxmem>128m</maxmem>
+          <excludes>
+             <exclude>**/os/solaris/*</exclude>
+             <exclude>**/os/windows/*</exclude>
+             <exclude>**/DCOMSandbox.java</exclude>
+             <exclude>**/DotNet.java</exclude>
+             <exclude>**/WindowsSlaveInstaller.java</exclude>
+             <exclude>**/WindowsInstallerLink.java</exclude>
+          </excludes>
+          <testExcludes>
+             <exclude>**/DCOMSandbox.java</exclude>
+          </testExcludes>
         </configuration>
       </plugin>
       <plugin>
